test/helpers/index.js

```diff
@@ -0,0 +1,23 @@
+const chance = require('chance')();
+const supertest = require('supertest');
+const app = require('../../app');
+const request = supertest(app);
+
+exports.createUsers = count => {
```

Comment:
Instead of use endpoint for create multiple users, you could use something like [factory-girl](https://github.com/aexmachina/factory-girl), this way seems like a lot of work and if /users endpoint changes behaviour this function should be changed each time